samples/pyodide/worker.mjs

```diff
@@ -0,0 +1,19 @@
+import { loadPyodide } from "pyodide:python";
+import worker from "./worker.py";
+
+export default {
```

Comment:
it is not very clear to me how much here is boiler plate and will need to be customized and why? Or will this file be in reality copy-pasted by everyone?

Reply:
Yeah, the plan is to generate whatever boilerplate is appropriate from config in followup work. This PR is just supposed to setup the implementation for that, and this `sample` is intended as an indication of how it works right now.